modules/nf-core/simpleaf/index/main.nf

```diff
@@ -0,0 +1,48 @@
+process SIMPLEAF_INDEX {
+    tag "$genome_fasta $transcript_fasta"
+    label 'process_high'
+
+    conda "bioconda::simpleaf=0.14.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/simpleaf:0.14.1--h4ac6f70_0':
+        'biocontainers/simpleaf:0.14.1--h4ac6f70_0' }"
+
+    input:
+    path genome_fasta
+    path genome_gtf
```

Comment:
```suggestion
    tuple val(meta), path(genome_fasta)
    tuple val(meta2), path(genome_gtf)
```

Meta-fy everything :p

Reply:
Ahh- we doing this even for reference-y things now? I'd missed that